src/app/static/src/app/components/downloadResults/UploadModal.vue

```diff
@@ -6,9 +6,11 @@
                 <div id="dataset-id" class="mt-4">
                     <span v-translate="'uploadFileDataset'"></span>
                     <span>{{ dataset }}</span></div>
+                <div class="pt-3 text-danger" id="output-file-error" v-if="outputFileError" v-translate="outputFileError"></div>
```

Comment:
No change needed here, but we should probably think about standardising these basic error messages. Would a component be overkill?

Reply:
You are right, since it's not a standard api error and no repetition as such. I don't see a point creating a component for it.